libcxx/docs/UserDocumentation.rst

```diff
@@ -237,22 +236,22 @@ C++26 Specific Configuration Macros
 Libc++ Extensions
 =================
 
-This section documents various extensions provided by libc++, how they're
-provided, and any information regarding how to use them.
+This section documents various extensions to the C++ standard library
+provided by libc++ and any information regarding how to use them.
```

Comment:
The old formulation was clearer. The new formulation reads as-if libc++ was "providing a C++ standard library".

Reply:
I don't understand why you marked this comment as resolved but didn't make any changes.